### ISSUE TITLE
Rename `master` to `main`

### ISSUE THREAD

**Issue by seanfcarroll** [activerecord-hackery/ransack#1174]
See 

- https://jarv.is/notes/github-rename-master/
- https://www.zdnet.com/article/github-to-replace-master-with-main-starting-next-month/

**Comment by yshmarov** [comment]
This is for new repositories. Old ones don't have to be renamed

**Comment by deivid-rodriguez** [comment]
@yshmarov I know we don't _need_ to do it, but I think @seanfcarroll is bringing this up as something that we _want_ to do.

**Comment by itsalongstory** [comment]
At the technical level, a lot of libraries use `master` as the default branch name, changing it to `main` may lead to unintended results.

```Gemfile
gem 'ransack', github: 'activerecord-hackery/ransack'
```

For example, in Gemfile, this will fetch from `master` branch, not `main` branch, so I think we don't need to do it.

**Comment by deivid-rodriguez** [comment]
Yes, I don't plan to change this at all unless we fix https://github.com/rubygems/rubygems/issues/4009 first and make sure that proper redirections are in place for the transition.

**Comment by seanfcarroll** [comment]
@deivid-rodriguez I see https://github.com/rubygems/rubygems/issues/4009 is closed, is it fixed now (wasn't clear to me in the issue).

**Comment by deivid-rodriguez** [comment]
Yes, this should be fixed now and released with bundler 2.2.5 (https://github.com/rubygems/rubygems/blob/3.2/bundler/CHANGELOG.md#225-january-11-2021). Obviously unless clients have upgraded to that version, they won't benefit from the fix.

**Comment by seanfcarroll** [comment]
> Obviously unless clients have upgraded to that version, they won't benefit from the fix.

Let's give it a few more months then.

**Comment by deivid-rodriguez** [comment]
Awesome, thanks @scarroll32!
